README.rst

```diff
@@ -188,57 +229,49 @@ to ``tornado``:
     app = connexion.App(__name__, specification_dir='swagger/')
     app.run(server='tornado', port=8080)
 
-You can use the Flask WSGI app with any WSGI container, e.g. `using Flask with uWSGI`_:
-
+You can use the Flask WSGI app with any WSGI container, e.g. `using Flask with uWSGI`_ (this is more common):
 .. code-block:: python
 
     app = connexion.App(specification_dir='swagger/')
     application = app.app # expose global WSGI application object
 
+Set up and run the installation code:
 .. code-block:: bash
 
     $ sudo pip3 install uwsgi
     $ uwsgi --http :8080 -w app -p 16  # use 16 worker processes
 
-You can run uWSGI with a large number of worker processes to get high concurrency.
-
 See the `uWSGI documentation`_ for more information.
 
 .. _using Flask with uWSGI: http://flask.pocoo.org/docs/latest/deploying/uwsgi/
 .. _uWSGI documentation: https://uwsgi-docs.readthedocs.org/
 
-Releasing Connexion
+Contributing to Connexion/TODOs
 ===================
+We welcome your ideas, issues, and pull requests—just follow the usual/standard practices.
 
-Build and upload new version to PyPI:
-
-.. code-block:: bash
-
-    $ ./release.sh <NEW-VERSION>
-
-License
--------
-Copyright 2015 Zalando SE
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-http://www.apache.org/licenses/LICENSE-2.0
+TODOs
+--------------
+If you'd like to become a more consistent contributor to Connexion, we'd love your help working on these:
+- Additional ways to handle OAuth 2 authentications
+- Silencing ImportErrors raised in modules referenced by operationId
+- Overriding default validation error message
```

Comment:
Maybe you can also include the link to the issues page or even the [waffle board](https://waffle.io/zalando/lizzy).